app/src/main/java/com/codingblocks/cbonlineapp/profile/ProfileActivity.kt

```diff
@@ -60,7 +56,7 @@ class ProfileActivity : BaseCBActivity() {
             }
             nameTv.text = "${it.firstname} ${it.lastname}"
             userNameTv.text = it.username
-            userImgView.loadSvg(it.photo ?: "")
+            userImgView.loadImage(it.photo ?: "", scale = true)
```

Comment:
Yup only this was to be done any guesses why ?

Reply:
Yeah that's am img url mostly jpg, can't load it as svg